examples/lighting-app/lighting-common/color_format/color_format.h

```diff
@@ -0,0 +1,45 @@
+/*
+ *
+ *    Copyright (c) 2020 Project CHIP Authors
+ *    All rights reserved.
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#pragma once
+
+#include <stdbool.h>
+#include <stdint.h>
+
+typedef struct RgbColor_
+{
+    uint8_t r;
+    uint8_t g;
+    uint8_t b;
+} RgbColor_t;
+
+typedef struct HsvColor_
+{
+    uint8_t h;
```

Comment:
Should h be `uint16_t` type instead of `uint8_t`? In the definition of `HsvToRgb`, the maximum value of this parameter is 360.

Reply:
In 3.2.2.2.1. Color Information Attribute Set specs - CurrentHue and CurrentSaturation are uint8 parameters. It can be 360, but in the formulas that are used, it is assumed that they are between 0 and 254.

Reply:
The CurrentHue Attribute value should multiply by 360/254 before passed to the function `HsvToRgb` according to the spec. This needs a `uint16_t` type `h`.
If you use the `uint8_t` type in this struct, the transform from CurrentHue to hue should be done in function `HsvToRgb`.

Reply:
Indeed, you are right. I will correct it.

Reply:
I checked and even if Hue = CurrentHue * 360 / 254 (i.e. the value needs to be transformed in degrees), HsvToRgb function takes a Hue in [0..254] range, therefore an inverse transformation would have to be applied. Therefore, we can use Hue as is (i.e. no transformation needed), and it can be transformed in degrees where needed.